paasta_tools/kubernetes_tools.py

```diff
@@ -146,6 +149,13 @@ class KubeService(NamedTuple):
     registrations: Sequence[str]
 
 
+class CustomResource(NamedTuple):
```

Comment:
is this actually `CustomResourceDefinition`?

Reply:
done